docs/contributing/server_administrator.rst

```diff
@@ -1,43 +1,39 @@
 ************************
 
-Once you have signed the NDA, you will be invited to the Jira board for the Infrastructure Working Group. From here, you can start to interact with the tasks which are outstanding and ask to be assigned any that you feel able to take on.
+Once you sign the NDA, the team invites you to the Jira board for the Infrastructure Working Group. From here, you can start to interact with outstanding tasks and request assignment to any that you feel able to take on.
 
 The board and issues are private due to the nature of the work.
 
 Work in the public domain
 *************************
 
-Unless there is significant reason not to, we default to being open and transparent. We work in the open, usually on our shared :xref:`Mautic Google Drive` folders. This ensures that if - for whatever reason - someone is unable to complete a task, it is easy for another contributor to pick up where they left off. It also means we can always find previous work that had been done if it needs to be re-used in the future.
+Unless there is significant reason not to, work should be open and transparent. The team documents tasks in :xref:`Mautic Google Drive` folders. This ensures that if - for whatever reason - someone is unable to complete a task, it's easy for another contributor to pick up where they left off. It also means you can find and re-use previous work in the future.
 
-As this team is often working with sensitive information, we default to a private-open approach. Please make sure that you do update your issues on the private Jira board and share any resources in the private Google Drive folder that might be relevant to the tasks you are working on.
+As this team is often working with sensitive information, work defaults to a private-open approach. Please make sure to update your issues on the private Jira board and share any resources in the private project folder that might be relevant to the tasks you are working on.
 
 Update regularly
 ****************
 
-Please make sure you provide regular updates on any tasks you are working on, and if at any point you're not going to be able to complete the task, please call that out by sending the Working Group Lead a message to inform them so that somebody else can pick it up. 
-
-We totally understand that life happens and it's easy to take on too much. No judgement at all. We try to be respectful of each other by ensuring we give as much notice as possible if we're not going to be able to fulfil a task assigned to us.
+Please provide regular updates on any tasks you are working on, and if at any point you're not going to be able to complete the task, please call that out by sending the Working Group Lead a message to inform them so that somebody else can pick it up. 
 
-.. vale on
\ No newline at end of file
+Life happens and it's easy to take on too much. There is no judgement. Being respectful of others means notifying the team early when you can't fulfill an assigned task.
```

Comment:
We can use the same one as the [Marketer page](https://github.com/mautic/mautic-community-handbook/pull/414/files#r2479606728).


```suggestion
Life can be unpredictable, and it’s easy to feel overwhelmed by trying to manage too much at once. There's no judgment here. The team's goal is to show respect for one another by providing as much notice as possible when unable to fulfill an assigned task.
```